pkgs/development/tools/electron/info.json

```diff
@@ -17,7 +17,7 @@
             "src": {
                 "fetcher": "fetchFromGitiles",
                 "hash": "sha256-lepGVhzXrBAY5YWwobe18FroRiOD/Q9f8QqazHDmvTY=",
-                "postFetch": "rm -r $out/third_party/blink/web_tests; rm -r $out/third_party/hunspell/tests; rm -r $out/content/test/data; rm -r $out/courgette/testdata; rm -r $out/extensions/test/data; rm -r $out/media/test/data; ",
+                "postFetch": "rm -r $out/third_party/blink/web_tests; rm -r $out/third_party/hunspell/tests; rm -r $out/content/test/data; rm -rf $out/courgette/testdata; rm -r $out/extensions/test/data; rm -r $out/media/test/data; ",
```

Comment:
chromium has since #357371 (specifically 8dd2f1add978a4747a5962f2874b8ad20f86b01c) more or less the same logic to resolve the dependencies.
But instead of having a bunch of `rm` in `postFetch`, chromium uses a custom `recompress = true` flag.

This is currently not available to electron as a consumer of the chromium derivation.
Well and it has its own downsides but oh well.

So nothing actionable so far, just as a fyi :)

https://github.com/NixOS/nixpkgs/blob/c2a355e310fb4f9ffd1d138cbb70579adfe4c483/pkgs/applications/networking/browsers/chromium/update.mjs#L93-L95

https://github.com/NixOS/nixpkgs/blob/878ced9a7f253495dcf34d38f73d65ae5dcfe0b9/pkgs/applications/networking/browsers/chromium/info.json#L24

https://github.com/NixOS/nixpkgs/blob/878ced9a7f253495dcf34d38f73d65ae5dcfe0b9/pkgs/applications/networking/browsers/chromium/common.nix#L154-L170

https://github.com/NixOS/nixpkgs/blob/878ced9a7f253495dcf34d38f73d65ae5dcfe0b9/pkgs/applications/networking/browsers/chromium/common.nix#L172-L186

(spoiler: it's `recompressTarball` all over again)